src/pages/NewChatPage.js

```diff
@@ -129,6 +131,23 @@ class NewChatPage extends Component {
         return sections;
     }
 
+    /**
+     * Attempts to find a report in onyx with the provided list of participants
+     * @param {Array} newParticipantList
+     * @returns {Array|undefined}
+     */
+    getChatByParticipants(newParticipantList) {
+        const sortedNewParticipantList = newParticipantList.sort();
+        return _.find(this.props.reports, (report) => {
+            // Chat rooms don't have participants, so we can skip them
+            if (!report || !report.participants) {
+                return false;
+            }
+            const sortedReportParticipants = report.participants.sort();
+            return sortedNewParticipantList.join() === sortedReportParticipants.join();
```

Comment:
Can we just use https://underscorejs.org/#isEqual instead of doing the sort and join? It will look much cleaner.

Reply:
+1